src/objective-c/BUILD

```diff
@@ -239,6 +239,18 @@ grpc_objc_library(
     ],
 )
 
+grpc_objc_library(
+    name = "grpc_objc_client_core_cronet_testing",
+    srcs = glob(["GRPCClient/private/GRPCCore/GRPCCoreCronet/*.m"]),
```

Comment:
Consider refactor this out to a separate target e.g. GRPCCoreCronet similar to how we structure our Cocoapod (https://github.com/grpc/grpc/blob/27e5b4e1877840b58c92d5d853c20ab713d19187/gRPC.podspec#L150) and have the test target depends on it instead

Reply:
There is already a `grpc_objc_client` at line 118.
The testing targets are configured differently from prod targets, I can take a look if they can be consolidated in later PRs.

Reply:
sg, let's do that in a follow up PR . The equivalent Pod test target is 

https://github.com/grpc/grpc/blob/ac585b8f7e78ea9d698bcc6c0019a336332b57e7/src/objective-c/tests/Podfile#L44